packages/xo-server-sdn-controller/src/index.js

```diff
@@ -424,52 +425,19 @@ class SDNController extends EventEmitter {
           )
         }
       }
-      for (const tunnel of tunnels) {
-        const accessPIF = xapi.getObjectByRef(tunnel.access_PIF)
-        if (accessPIF.host !== host.$ref) {
-          continue
-        }
-
-        const poolNetwork = find(this._poolNetworks, {
-          network: accessPIF.network,
-        })
-        if (poolNetwork == null) {
-          continue
-        }
-
-        if (accessPIF.currently_attached) {
-          continue
-        }
+    }
+  }
 
-        log.debug(
-          `Pluging PIF: '${accessPIF.device}' for host: '${host.name_label}' on network: '${accessPIF.$network.name_label}'`
-        )
-        try {
-          await xapi.call('PIF.plug', accessPIF.$ref)
-        } catch (error) {
-          log.error(
-            `XAPI error while pluging PIF: '${accessPIF.device}' on host: '${host.name_label}' for network: '${accessPIF.$network.name_label}'`
-          )
-        }
+  async _hostMetricsUpdated(hostMetrics) {
+    const ovsdbClient = find(this._ovsdbClients, {
```

Comment:
What if there aren't any?

Reply:
Must be since I create a `ovsdbClient` for each hosts of the managed pools.